StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentMethodFormViewController.swift

```diff
@@ -220,11 +220,13 @@ extension PaymentMethodFormViewController {
         }()
 
         let linkMode = elementsSession.linkSettings?.linkMode
+        let billingAddress = instantDebitsFormElement?.billingAddress
```

Comment:
It seems like we’re sending the address even if `attachDefaults` is set to `false` and address collection is set to `never`. Can you take another look?